src/platform/plugins/shared/workflows_management/common/schema.ts

```diff
@@ -69,6 +69,15 @@ const connectors = [
       },
     ],
   },
+  {
```

Comment:
If you declare custom step type in `@kbn/workflows/spec`, there's no need to specify it here.